src/main/drivers/display.c

```diff
@@ -214,6 +221,14 @@ uint16_t displayTxBytesFree(const displayPort_t *instance)
     return instance->vTable->txBytesFree(instance);
 }
 
+bool displayFontMetadata(displayFontMetadata_t *metadata, const displayPort_t *instance)
```

Comment:
Maybe `displayGetFontMetadata`?

Reply:
Just this function or also the field in the vtable?

Reply:
The field as well - let's make names self-explanatory.